dts/bindings/clock/nxp,lpc-syscon.yaml

```diff
@@ -14,5 +14,17 @@ properties:
   "#clock-cells":
     const: 1
 
+  i3c-clk-divider:
+    type: array
+    description: Main clock divider for each I3C instance
+
+  i3c-clk-divider-tc:
+    type: array
+    description: TC clock divider for each I3C instance
+
+  i3c-clk-divider-slow:
+    type: array
+    description: Slow clock divider for each I3C instance
+
```

Comment:
i dont think these belong in the syscon binding either, eventually the clock management framework being proposed should handle this clock as it's own node in the clock tree. Notice there is nothing else in this binding right now related to clock dividers. I dont think we want to go down this route and create more cleanup and have to change bindings in the future.

right now I think best option is what we already do for soc specific system clocking configuration which is just to use soc specific kconfigs.

Reply:
How about I leave this binding in the `i3c` node and we handle this when the clocking framework is in place instead of adding `Kconfigs` for each SoC we add.